build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -29,11 +29,17 @@
       <DotNetPath>$(Stage0Path)</DotNetPath>
 
       <BaseOutputDirectory>$(RepoRoot)/artifacts/$(Rid)</BaseOutputDirectory>
-      <OutputDirectory>$(BaseOutputDirectory)/stage2/</OutputDirectory>
+      <ToolsOutputDirectory>$(BaseOutputDirectory)/tools</ToolsOutputDirectory>
+      <Stage1Directory>$(BaseOutputDirectory)/stage1</Stage1Directory>
+      <Stage1SymbolsDirectory>$(BaseOutputDirectory)/stage1symbols</Stage1SymbolsDirectory>
+      <DotnetStage1>$(Stage1Directory)/dotnet.exe</DotnetStage1>
+      <Stage2Directory>$(BaseOutputDirectory)/stage2</Stage2Directory>
```

Comment:
Didn't notice that. I will fix that later then. I am on the road. I guess I will fix this and the inputs for the archiver and then i will merge the pr.